compiler-rt/lib/mpk_untrusted/tests/mpk-key/src/mpk_key.c

```diff
@@ -0,0 +1,92 @@
+#define _GNU_SOURCE
+#define HAS_MPK 1
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/mman.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+
+static inline void __wrpkru(unsigned int pkru) {
+  unsigned int eax = pkru;
+  unsigned int ecx = 0;
+  unsigned int edx = 0;
+  asm volatile(".byte 0x0f,0x01,0xef\n\t" : : "a"(eax), "c"(ecx), "d"(edx));
+}
+
+int __pkey_set(int pkey, unsigned long rights, unsigned long flags) {
+  unsigned int pkru = (rights << (2 * pkey));
+  __wrpkru(pkru);
+  return 1;
+}
+
+int __pkey_mprotect(void *ptr, size_t size, unsigned long orig_prot,
+                    unsigned long pkey) {
+  return syscall(SYS_pkey_mprotect, ptr, size, orig_prot, pkey);
+}
+
+int __pkey_alloc(void) { return syscall(SYS_pkey_alloc, 0, 0); }
+
+int __pkey_free(unsigned long pkey) { return syscall(SYS_pkey_free, pkey); }
+
+#define errExit(msg)                                                           \
+  do {                                                                         \
+    perror(msg);                                                               \
+    exit(EXIT_FAILURE);                                                        \
+  } while (0)
+
```

Comment:
can you run clang format, or clang-format-patch on your PR? At least on the code that is only written by us, and not in an exising LLVM/Clang file.